REMAINING.PUBLIC.md

```diff
@@ -0,0 +1,70 @@
+### Remaining public aliases
+
+annotation_reader
+cache.app_clearer
+$commandId
+debug.templating.engine.php
+profiler.request_matcher
+router
+security.access.decision_manager
+security.acl.cache
+security.acl.dbal.connection
+security.acl.provider
+security.encoder_factory
+security.firewall
+security.password_encoder
+session.handler
+session.storage
+session.storage.filesystem
+templating
+templating.loader
+translator
+twig.loader
```

Comment:
even this one should become private IMO. There is no reason to access it dynamically at runtime IMO.